config.md

```diff
@@ -210,7 +210,7 @@
 |idleTimeout|The max duration to hold a HTTP keepalive connection between calls|[`time.Duration`](https://pkg.go.dev/time#Duration)|`475ms`
 |maxIdleConns|The max number of idle connections to hold pooled|`int`|`100`
 |method|The HTTP Method to use when invoking the Gas Oracle REST API|`string`|`GET`
-|mode|The gas oracle mode|connector | restapi | disabled|`connector`
+|mode|The gas oracle mode - connector, restapi, fixed, or disabled|`string`|`connector`
```

Comment:
This one is here: https://github.com/hyperledger/firefly-transaction-manager/blob/f319338146cf44c39695497f8f3f17a9169225d0/internal/tmmsgs/en_config_descriptions.go#L57

Reply:
https://github.com/hyperledger/firefly-transaction-manager/pull/39